lib/linux.coffee

```diff
@@ -31,4 +31,5 @@ exports.isSystem = (drive, callback) ->
 			return callback(false)
 
 		result = tableParser.parse(stdout)
-		return callback(result[0].RM?[0] isnt '1')
+		rmFlag = result[0].RM?[0] or result[0]['MAJ:MIN'][1]
```

Comment:
@Page- There seems to be an edge case where the table parser module I'm using interprets the RM value as the second value of MAJ:MIN. I'll add a comment explaining that.